providers/config/providerConfig.go

```diff
@@ -38,6 +56,15 @@ func LoadProviderConfigs(fname string) (map[string]map[string]string, error) {
 	return results, nil
 }
 
+func executeCredsFile(filename string) ([]byte, error) {
+	pwd, err := os.Getwd()
```

Comment:
Is prepending `os.Getwd()` really required?

It would be more simple to prepend "./"

```
        cmd := filename
        if ! strings.HasPrefix(fname, "/") {
                // if the path doesn't start with `/` make sure we aren't relying on $PATH.
               cmd = filepath(".", filename)
        }
	out, err := exec.Command(cmd).Output()
        return out
```